source/conda.rst

```diff
@@ -169,6 +169,43 @@ JupyterHub.
     (This often trips up Python users, because Python tolerates trailing commas
     in lists.)
 
+4. OPTIONAL : Specifying environment variables
+
+   In some cases, you may want to specify some environment variables before the
+   ipython kernel is run. Add them to the "argv" list to specify them.
+   For example, for hdf5 images saved by the EIGER detector, the lz4
+   compression plugin is required. The plugin is located by the hdf5 library
+   through the environment variable:
+
+    .. code-block:: bash
+
+        HDF5_PLUGIN_PATH=/path/to/HDF5Plugin
+
+   This is added by simply adding this line above to the "argv" field:
+
+    .. code-block:: json
+
+        {
+         "argv": [
+         "HDF5_PLUGIN_PATH=/path/to/HDF5Plugin",
```

Comment:
This should ideally be a path into the enviroment!

Reply:
so `/home/dallan/conda_envs/myenv/path/to/HDF5Plugin` then?
Also, added a note to make it clear that this should precede the executables. this is like running
`ENV1=123 ENV2=234 myexec`